src/host/CommandListPopup.cpp

```diff
@@ -50,7 +50,7 @@ static COORD calculatePopupSize(const CommandHistory& history)
     return { gsl::narrow<short>(width), height };
 }
 
-CommandListPopup::CommandListPopup(SCREEN_INFORMATION& screenInfo, const CommandHistory& history) :
+CommandListPopup::CommandListPopup(SCREEN_INFORMATION& screenInfo, CommandHistory& history) :

```

Comment:
This is missing the const it had previously

Reply:
ok